src/components/DateTemperature/index1.js

```diff
@@ -0,0 +1,45 @@
+export const MainBlockDay = ({ data, i, j, title }) => {
+	const temp = (n) => {
```

Comment:
if u use the same func in other folders move it to utils/helpers/get-temperature.js for example